cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinContributor.java

```diff
@@ -5,6 +5,7 @@
 import static no.unit.nva.cristin.lambda.constants.MappingConstants.NVA_API_DOMAIN;
 import static no.unit.nva.cristin.lambda.constants.MappingConstants.PERSON_PATH;
 import static no.unit.nva.cristin.lambda.constants.MappingConstants.SHOULD_CREATE_CONTRIBUTOR_ID;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
Is the actual mapping to a NVA contributor coming in a different PR?

Reply:
There is already mapping from CristinContributor to Contributor, I have only added this method to extract verificationStatus                                 `.withVerificationStatus(extractVerificationStatus())`

```
    public Contributor toNvaContributor() {
        String fullName = constructFullName();
        Identity identity = new Identity.Builder()
                                .withName(fullName)
                                .withId(constructId())
                                .withVerificationStatus(extractVerificationStatus())
                                .build();

        return new Contributor.Builder()
                   .withIdentity(identity)
                   .withCorrespondingAuthor(false)
                   .withAffiliations(extractAffiliations())
                   .withRole(extractRoles())
                   .withSequence(contributorOrder)
                   .build();
    }
```